e2e/cypress/integration/ui/settings/team_mgmt.spec.ts

```diff
@@ -52,14 +54,16 @@ describe('team management', () => {
 
   afterEach(() => {
     cy.saveStorage();
-    cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['teams']);
   });
 
   after(() => {
-    cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['projects']);
+    cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'teams']);
   });
 
-  describe('no custom initial page state', () => {
+  describe('teams list page', () => {
+    const dropdownNameUntilEllipsisLen = 25;
+    const projectDropdown = 'app-team-management app-projects-dropdown ';
```

Comment:
The trailing space here is vital but that is not at all obvious. Consider:
```suggestion
    const projectDropdown = 'app-team-management app-projects-dropdown' + ' '; // trailing space vital!
```

Reply:
Some ideas to make it more obvious:

- let's name the constant after what it is -- a css selector (prefix)
- let's add the space in its call sites (?)
- let's explain why it's "vital"